src/PresentationalComponents/CreateImageWizard/ImageWizardFooter.js

```diff
@@ -14,24 +14,33 @@ const ImageWizardFooter = (props) => {
                     {({ activeStep, onNext, onBack, onClose }) => {
                         let nextButtonText = 'Next';
                         if (activeStep.name === 'Review') {
-                            nextButtonText = props.disable ? 'Creating...' : 'Create';
+                            nextButtonText = props.isSaveInProgress ? 'Creating...' : 'Create';
+                        }
+
+                        let nextButtonIsDisabled = false;
```

Comment:
nitpicky, but you can avoid the conditional on the next line by just `let nextButtonIsDisabled = props.isSaveInProgress;`

Reply:
Fixed